macaroonbakery/codec.py

```diff
@@ -278,7 +278,7 @@ def encode_uvarint(n, data):
         data.append(this_byte | 128)
 
 
-def _decode_uvarint(data):
+def decode_uvarint(data):
```

Comment:
I've just realised that this succeeds incorrectly on badly format varints.
For example, `decode_uvarint(b'\xff')` returns (127, 1) even though the varint
is not terminated correctly. It also returns (0, 0) on an empty sequence, which
doesn't seem right either.

I'd suggest fixing that in a separate PR.